docs/dictionary.rst

```diff
@@ -14,8 +14,11 @@ Universal domain names
 * Nazev stranky / rubriky = Page Heading
 * Nickname = nick (always shortened)
 * Anotace = synopsis
-* Prispevek = Creation
+* Dílo = Creation
+* Diskuze pod díly = Discussion
```

Comment:
Just of note, I think the term should be only "Diskuze" and "pod dily" will be implied by that